### PR TITLE
Add 3.0.0-rc1 to changelog

### PR DESCRIPTION
**Pull request by vweevers**
😅

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,73 @@
...
+* Remove obsolete `#_approximateSize` (@vweevers)
+* Remove obsolete `#_isBuffer` (@vweevers)
+* Remove obsolete `testBuffer` from abstract tests (@vweevers)
+* Remove obsolete writestream test from `test-levelup.js` (@vweevers)
+* Rely on `abstract-leveldown` defaults in `Iterator` constructor (@vweevers)
+* Rely on `abstract-leveldown` callback defaults (@vweevers)
+* Remove testling from `package.json` (@vweevers)
+* Remove `level.js` logo (@vweevers)
+
+**Historical Note** As a result of removing `IDBWrapper`, only modern browsers with a non-prefixed `window.indexedDB` are supported in this release. IE10 and Firefox 52 are known to be broken. The current test matrix of `level-js` includes the latest versions of Chrome, Firefox, Safari, Edge and IE.
```
Comment (vweevers) [inline CHANGELOG.md]: We can later move (some or all of) these notes to the upgrade guide

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,73 @@
 
 ## [Unreleased]
 
+## [3.0.0-rc1] - 2016-05-26
+
+## Changed
+* Upgrade `abstract-leveldown` from `0.12.0` to `5.0.0` (@vweevers)
```
Comment (ralphtheninja) [inline CHANGELOG.md]: Aah nice to use _from_ and _to_. We should have this format in all our changelogs.

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,73 @@
 
 ## [Unreleased]
 
+## [3.0.0-rc1] - 2016-05-26
+
+## Changed
```
Comment (vweevers) [inline CHANGELOG.md]: Wrong level. Fixing

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,73 @@
...
+* Make real `tape` tests out of `test-levelup.js` (@vweevers)
+* Restructure custom tests to follow abstract test suite format (@vweevers)
+
+### Added
+* Add continuous browser tests with `airtap` and Sauce Labs (@vweevers)
+* Add `prefix` and `version` options to constructor (@vweevers)
+* Detect binary key support and fallback to `String(buffer)` (@vweevers)
+* Detect array key support and fallback to `String(array)` (@vweevers)
+* Test all value types of the structured clone algorithm (@vweevers)
+* Catch `DataCloneError` if the environment does not support serializing the type of a key or value
```
Comment (ralphtheninja) [inline CHANGELOG.md]: Missing (@vweevers)

File: CHANGELOG.md (hunk 2 above)
Comment (vweevers) [reply CHANGELOG.md]: Sure! I figured it was important here because we skipped a few majors :)

**Comment by vweevers** [comment]
I'll copy it to the GH release.

**Pull request merged**